Sources/DiscordModels/Types/Gateway.swift

```diff
@@ -477,6 +477,19 @@ public struct Gateway: Sendable, Codable {
                 self.status = status
                 self.afk = afk
             }
+
+            public func encode(to encoder: any Encoder) throws {
+                var container = encoder.container(keyedBy: CodingKeys.self)
+                if let since {
+                    try container.encode(since, forKey: .since)
+                } else {
+                    try container.encodeNil(forKey: .since)
+                }
+                try container.encodeIfPresent(self.since, forKey: .since)
```

Comment:
I think you forgot to remove this line 😅

Reply:
Oh, yeap, my bad) fixed